web/templates/popovers/left_sidebar/left_sidebar_views_popover.hbs

```diff
@@ -0,0 +1,7 @@
+<div class="popover-menu" data-simplebar data-simplebar-tab-index="-1">
+    <ul role="menu" class="popover-menu-list condensed-views-popover-menu">
+        {{#each views}}
+            {{> left_sidebar_view_popover_item .}}
+        {{/each}}
+    </ul>
+</div>
```

Comment:
A technique for testing this that you may find helpful is that you can use your browser's developer tools and "Edit HTML" on a section of the DOM tree, copy that to a file, and diff the result pre-patch vs post-patch.